src/main/resources/templates/fragments/reusable-components.html

```diff
@@ -1,43 +1,59 @@
-        <img src="https://s3.drafthouse.com/ui/sidebar-placeholder-blank.png" class="card-img-top" alt="placeholder image">
-        <div class="card-body text-dark">
-            <h5 class="card-title">Collection title</h5>
-            <p class="card-text">Sample text for the collection description</p>
-            <a href="#" class="btn btn-primary">View Collection</a>
+    <div th:fragment="viewcollectioncard">
+        <div class="card" style="width: 18rem;border-style: solid;border-width: medium;border-color: grey;">
+            <img src="https://s3.drafthouse.com/ui/sidebar-placeholder-blank.png" class="card-img-top" alt="placeholder image">
+            <div class="card-body text-dark">
+                <h5 class="card-title">Collection title</h5>
+                <p class="card-text">Sample text for the collection description</p>
+                <a href="#" class="btn btn-primary">View Collection</a>
+            </div>
         </div>
     </div>
-</div>
 
-<div th:fragment="addcollectioncard">
-    <div class="card" style="width: 18rem;border-style: solid;border-width: medium;border-color: grey;">
-        <div class="card-body text-dark">
-            <h5 class="card-title">Start Here</h5>
-            <p class="card-text">Click below to add a new collection</p>
-            <a href="#" class="btn btn-primary">Add Collection</a>
+    <div th:fragment="addcollectioncard">
+        <div class="card" style="width: 18rem;border-style: solid;border-width: medium;border-color: grey;">
+            <div class="card-body text-dark">
+                <h5 class="card-title">Start Here</h5>
+                <p class="card-text">Click below to add a new collection</p>
+                <a href="#" class="btn btn-primary">Add Collection</a>
+            </div>
         </div>
     </div>
-</div>
 
-<div th:fragment="itemcard">
-    <div class="card" style="width: 18rem;border-style: solid;border-width: medium;border-color: grey;">
-        <img class="card-img-top" src="https://s3.drafthouse.com/ui/sidebar-placeholder-blank.png" alt="placeholder image">
-        <div class="card-body text-dark">
-            <h5 class="card-title">Item title</h5>
-            <p class="card-text">Sample text for the item description</p>
-            <a href="#" class="btn btn-primary">Edit Item</a>
+    <div th:fragment="itemcard">
+        <div class="card" style="width: 18rem;border-style: solid;border-width: medium;border-color: grey;">
+            <img class="card-img-top" src="https://s3.drafthouse.com/ui/sidebar-placeholder-blank.png" alt="placeholder image">
+            <div class="card-body text-dark">
+                <h5 class="card-title">Item title</h5>
+                <p class="card-text">Sample text for the item description</p>
+                <a href="#" class="btn btn-primary">Edit Item</a>
+            </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+</body>
```

Comment:
I added the layout-design theme to this webpage to make it have a better review, and I also re-formatted the code.
Sources:
https://www.w3schools.com/html/html_layout.asp
https://www.baeldung.com/java-clean-code
https://www.tutorialspoint.com/java/java_documentation.htm